src/main/webapp/app/course/competencies/components/course-competency-relation-form/course-competency-relation-form.component.ts

```
@@ -0,0 +1,289 @@
+            this.isLoading.set(false);
+        }
+    }
+
+    protected async deleteRelation(): Promise<void> {
+        try {
+            this.isLoading.set(true);
+            const deletedRelation = this.relations().find(
+                ({ headCompetencyId, tailCompetencyId, relationType }) =>
+                    headCompetencyId == this.headCompetencyId() && tailCompetencyId == this.tailCompetencyId() && relationType === this.relationType(),
+            );
+            await this.courseCompetencyApiService.deleteCourseCompetencyRelation(this.courseId(), deletedRelation!.id!);
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)